pkg/scheduler/controller/common/seed.go

```diff
@@ -27,7 +27,9 @@ func VerifySeedReadiness(seed *gardencorev1beta1.Seed) bool {
 	if cond := gardencorev1beta1helper.GetCondition(seed.Status.Conditions, gardencorev1beta1.SeedGardenletReady); cond == nil || cond.Status != gardencorev1beta1.ConditionTrue {
 		return false
 	}
-
+	if cond := gardencorev1beta1helper.GetCondition(seed.Status.Conditions, gardencorev1beta1.SeedSystemComponentsHealthy); cond == nil || cond.Status != gardencorev1beta1.ConditionTrue {
```

Comment:
This doesn't seem to be resolved: https://github.com/gardener/gardener/pull/5274/files#r793830880